packages/autorest.python/autorest/codegen/models/parameter.py

```diff
@@ -92,6 +92,10 @@ def __init__(
         )
         self.hide_in_method: bool = self.yaml_data.get("hideInMethod", False)
 
+    @property
```

Comment:
there already is a property for this behavior, `hide_in_method`

Reply:
`hide_in_method` is to hide some special headers in both `builder` and `operation`. However, `maxpagesize` is only hided in `operation` but keep in `builder`